src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Financial/Application_ReadOnly.cshtml

```diff
@@ -102,68 +98,89 @@
-                    </dd>
-                </div>
-                <div class="govuk-summary-list__row">
-                    <dt class="govuk-summary-list__key">
-                        Financial health assessed by
-                    </dt>
-                    <dd class="govuk-summary-list__value">
-                        @Model.FinancialReviewDetails.GradedBy
-                    </dd>
-                </div>
-                @if (!string.IsNullOrEmpty(Model.FinancialReviewDetails.Comments))
-                {
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">
+                                Uploaded @pluraliseFiles
+                            </dt>
+
+                            <dd class="govuk-summary-list__value">
+                            @if (numberOfFiles == 0)
+                            {
+                                <text>N/A</text>
+                            }
+                            else
+                            {
+                                <ul class="govuk-list">
+                                    @foreach (var file in Model.FinancialReviewDetails?.ClarificationFiles)
+                                    {
+                                        <li><a class="govuk-link govuk-link--no-visited-state" asp-action="DownloadClarificationFile" asp-controller="RoatpFinancial" asp-route-ApplicationId="@Model.ApplicationId" asp-route-Filename="@file.Filename">@file.Filename</a></li>
+                                    }
+                                </ul>
+                            }
+                        </div>
+                    }
                     <div class="govuk-summary-list__row">
-                    @if (inClarificationMode)
-                    {
                         <dt class="govuk-summary-list__key">
-                            Internal comments
+                            Outcome
                         </dt>
-                    }
-                    else
-                    {
+                        <dd class="govuk-summary-list__value">
+                            @Model.FinancialReviewDetails.SelectedGrade
+
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
                         <dt class="govuk-summary-list__key">
-                            Comments
+                            Financial health assessed on
                         </dt>
-                    }
-                    <dd class="govuk-summary-list__value">
-                            @Model.FinancialReviewDetails.Comments
+                        <dd class="govuk-summary-list__value">
+                            @if (Model.FinancialReviewDetails.GradedDateTime.HasValue)
+                            {
+                                @Html.Raw(Model.FinancialReviewDetails.GradedDateTime.Value.ToLongDateString())
+                            }
```

Comment:
Just noticed this but shouldn't this be: `@Model.FinancialReviewDetails.GradedDateTime.ToSfaShortDateString()` ?

May need to include: `@using SFA.DAS.AdminService.Common.Extensions`

Currently, when I use American English, this displays differently 😈

Reply:
not sure, I'll try it

Reply:
yes, will do